old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/06-CRS-formats-proj4-EPSG.Rmd

```diff
@@ -166,45 +224,114 @@ Note that there are no specified units above. This is because this geographic
 coordinate reference system is in latitude and longitude which is most
 often recorded in *Decimal Degrees*.
 
-<i class="fa fa-star"></i> **Data Tip:** the last portion of each `proj4` string
+<i class="fa fa-star"></i> **Data Tip:** the last portion of each `PROJ4` string
 is `+towgs84=0,0,0 `. This is a conversion factor that is used if a `datum`
 conversion is required. We will not deal with datums in this particular series.
-{: .notice}
-
-
-* Read more about <a href="https://www.nceas.ucsb.edu/scicomp/recipes/projections" target="_blank">all three formats from the National Center for Ecological Analysis and Synthesis.</a>
 
-* A handy four page overview of CRS <a href="https://www.nceas.ucsb.edu/~frazier/RSpatialGuides/OverviewCoordinateReferenceSystems.pdf" target="_blank">including file formats on page 1.</a>
 
-#### EPSG codes
-The EPSG codes are a structured dataset of CRSs that are used around the world. They were
+## EPSG codes
+The EPSG codes are a structured dataset of `CRS`s that are used around the world. They were
 originally compiled by the, now defunct, European Petroleum Survey Group, hence the
-EPSG acronym. Each code is a four digit number.  
+EPSG acronym. Each code is a four or five digit number.  
+ 
+<a href="http://spatialreference.org/" target="_blank">Spatialreference.org</a> is 
+a great tool to look up `CRS` strings using an EPSG code. Use the search to find 
+an EPSG code and the associated `CRS` of interest. You will then see the various 
+formats associated with that `CRS`.
 
 The codes and more information can be found on these websites:
-* <a href="http://www.epsg-registry.org/" target="_blank">The EPSG registry. </a>
-* <a href="http://spatialreference.org/" target="_blank">Spatialreference.org</a>
-* <a href="http://spatialreference.org/ref/epsg/" target="_blank">list of ESPG codes.</a>
+* <a href="http://www.epsg-registry.org/" target="_blank">The EPSG registry</a>
+* <a href="http://spatialreference.org/ref/epsg/" target="_blank">Spatial Reference's list of EPSG codes</a>
 
-``` {r crs-strings}
+You can also create a list of EPSG codes using the` make_epsg()` function in 
+the `rgdal` package in `R`. This can be useful if you need to look up a code 
+without using <a href="http://spatialreference.org/" target="_blank">Spatialreference.org</a>.
 
+``` {r EPSGcode-list}
+#load library
 library('rgdal')
```

Comment:
I think this is redundant, as you have loaded 'rgdal' above

Reply:
Thank you for catching that!